config/crds/apis.kcp.dev_apibindings.yaml

```diff
@@ -92,17 +92,18 @@ spec:
                               metadata.namespace field. If "name" is unset, all objects
                               from the namespace are being claimed.
                             type: string
-                        required:
-                        - namespace
                         type: object
+                        x-kubernetes-validations:
+                        - message: either "namespace" or "name" and "namespace" must
+                            be set
+                          rule: has(self.__namespace__) || has(self.name) && has(self.__namespace__)
```

Comment:
isn't it the same as "namespace must be set" ? `A or (A and B)` is the same as `A`.

Reply:
fixed, using a different expression to allow both `name` and `namespace`